src/dev/ci_setup/setup_env.sh

```diff
@@ -180,6 +180,15 @@ fi
 ###
 cp -f "$KIBANA_DIR/src/dev/ci_setup/.bazelrc-ci" "$HOME/.bazelrc";
 
+###
+### remove write permissions on buildbuddy remote cache for prs
```

Comment:
I assume this doesn't actually modify or remove the permissions, right?
```suggestion
### prs don't have write permissions on buildbuddy remote cache
```

Reply:
It is actually adding a flag into the `bazelrc` file for ci in order to not allow to upload results to the remote cache on those 😄

Reply:
Hmmm, is that something we can enforce with different tokens for PR jobs or something?